### PR TITLE
update to publish from new frontend

### PR DESCRIPTION
**Pull request by jrhite**

### CODE REVIEW

File: client/packages/react-app/build/service-worker.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * The workboxSW.precacheAndRoute() method efficiently caches and responds to
+ * requests for URLs in the manifest.
+ * See https://goo.gl/S9QRab
+ */
+self.__precacheManifest = [].concat(self.__precacheManifest || []);
+workbox.precaching.precacheAndRoute(self.__precacheManifest, {});
+
+workbox.routing.registerNavigationRoute(workbox.precaching.getCacheKeyForURL("./index.html"), {
+  
+  blacklist: [/^\/_/,/\/[^/?]+\.[^/]+$/],
```
Comment (Pongch) [inline client/packages/react-app/build/service-worker.js:38]: this stuff is autogenerated right?

File: client/packages/react-app/build/static/media/circle.2d975615.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22C17.5228 22 22 17.5228 22 12C22 9.27455 20.9097 6.80375 19.1414 5" stroke="#2F80ED" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (Pongch) [inline client/packages/react-app/build/static/media/circle.2d975615.svg:3]: this is react-specific SVGs right? shall we remove this?

File: client/packages/react-app/build/static/media/x.b0200bf7.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-x"><line x1="18" y1="6" x2="6" y2="18"></line><line x1="6" y1="6" x2="18" y2="18"></line></svg>
```
Comment (Pongch) [inline client/packages/react-app/build/static/media/x.b0200bf7.svg:1]: shall we remove this file too?

**Review by Pongch — APPROVED** [review]
lgtm, just need to remove the boiler-plate react SVGs

File: client/packages/react-app/build/service-worker.js (hunk 1 above)
Comment (jrhite) [reply client/packages/react-app/build/service-worker.js:38]: yes. all files in the the `build` dir are auto-generated.

File: client/packages/react-app/build/static/media/circle.2d975615.svg (hunk 2 above)
Comment (jrhite) [reply client/packages/react-app/build/static/media/circle.2d975615.svg:3]: this is all auto-generated and happens somewhere within react-scripts (created by `create-eth-app` (a variant of `create-react-app` which is what @nichanank used).

not sure where it gets created, but we can add as a todo item for later if desired.

**Pull request merged**